habitat-lab/habitat/tasks/nav/nav.py

```diff
@@ -1376,6 +1417,68 @@ def _apply_velocity_action(
 
         return final_agent_state
 
+    def _apply_camera_pitch_velocity_action(
+        self,
+        camera_pitch_angular_velocity: float,
+        time_step: Optional[float] = None,
+    ):
+        """
+        Apply velocity command to the camera tilt angle for looking up and down
+        """
+        if time_step is None:
+            time_step = self._time_step
+
+        # Map velocity actions
+        self.vel_control.linear_velocity = np.array([0.0, 0.0, 0.0])
+        self.vel_control.angular_velocity = np.array(
+            [camera_pitch_angular_velocity, 0.0, 0.0]
+        )
+        sensor_states = self._sim.agents[0]._sensors  # type: ignore
+        for sensor in sensor_states:
+            sensor_state = self._sim.agents[0]._sensors[sensor].node  # type: ignore
```

Comment:
I dont understand what is happening here. If we are going to break the loop after the first step why have the for loop at all?

Reply:
Thank you! I directly use key of dict now